src/Appwrite/Utopia/Response/Model/Project.php

```diff
@@ -130,6 +130,12 @@ public function __construct()
                 'example' => new stdClass,
                 'array' => true,
             ])
+            ->addRule('services', [
```

Comment:
Not sure about this structure, I would try to avoid nested objects, maybe we can have a key per service where the default value is `true`?